docs/api/crash-reporter.md

```diff
@@ -58,6 +58,17 @@ sent or the crash reporter has not been started, `null` is returned.
 Returns all uploaded crash reports. Each report contains the date and uploaded
 ID.
 
+### `crashReporter.setExtraParameters(extra)`
```

Comment:
So these are meant to completely replace the `extra` parameters specified to `start` right? Maybe the docs should mention something like `This will override any extra parameters specified to start()` just to make things clear.